webhook/forward.go

```diff
@@ -179,7 +181,7 @@ func forwardEvent(port int, ev wsEventReceived) (*httpEventForward, error) {
 		fmt.Printf("%s\n", ev.Body)
 		return &httpEventForward{Status: 200, Header: make(http.Header), Body: []byte("OK")}, nil
 	}
-	webhookRcvServerURL := fmt.Sprintf("http://localhost:%d", port)
+	webhookRcvServerURL := fmt.Sprintf("http://localhost:%d%s", port, path)
```

Comment:
Ensures there's always a slash at the beginning of path: 

```suggestion
	webhookRcvServerURL := fmt.Sprintf("http://localhost:%d/%s", port, strings.TrimPrefix(path, "/"))
```